charts/alfresco-search-enterprise/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v2
 name: alfresco-search-enterprise
 description: A Helm chart for deploying Alfresco Elasticsearch connector
 type: application
-version: 4.4.0
+version: 4.4.1
```

Comment:
more a (missing) feature than a bugfix

Reply:
To a certain extend this is a bug. It's a so common use case (in. particular for this components which talks to different 3rd party services) that I decided to make it a bug in both github and JIRA, hence the version